CONTRIBUTING.md

```diff
@@ -145,6 +145,57 @@ feat(pencil): add 'graphiteWidth' option
 perf(pencil): remove graphiteWidth option
 ```
 
+## Becoming a Maintainer
```

Comment:
There is a table of contents at the top of the document. Please add a link to this section.

See the commits link for reference.

## <a name="commits"></a> Git Commit Guidelines